go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/felixge/fgprof v0.9.3
 	github.com/getsentry/sentry-go v0.22.0
 	github.com/go-errors/errors v1.4.2
-	github.com/go-git/go-git/v5 v5.8.1
+	github.com/go-git/go-git/v5 v5.8.0
```

Comment:
looks like you downgraded go-git, can you revert that?

Reply:
oh idk how that happened